Invoices.elm

```diff
@@ -24,45 +25,50 @@ type alias Product =
     , unit : Int
     , tax : Float
     , price : Float
-    , total : Float
     }
 
 
 type alias Model =
-    { total : Float
-    , products : List Product
+    { products : List Product
     }
 
 
 initModel : Model
 initModel =
-    Model 0 []
+    Model []
 
 
-createProduct : Int -> Product
-createProduct id =
-    Product id "" 1 21 0.0 0.0
+addNew : Model -> Model
+addNew { products } =
+    products
+        ++ [ { id = List.length products, item = "", unit = 1, tax = 21, price = 0 } ]
```

Comment:
seems to be safer to generate id from the state rather than to have a function which requires the user of this function to define which id is right one.